src/mudlet-lua/lua/geyser/GeyserAdjustableContainer.lua

```diff
@@ -1,28 +1,29 @@
 --Adjustable Container
---Just use it like a normal Geyser Container with some extras like: 
+--Just use it like a normal Geyser Container with some extras like:
 --moveable, adjustable size, attach to borders, minimizeable, save/load...
 --right click on top border for menu
 --Inspired heavily by Adjustable Label (by Jor'Mox ) and EMCO (by demonnic )
 --by Edru 2020
 
 Adjustable = Adjustable or {}
 
---- Adjustable Container
--- @class table
--- @name Adjustable.Container
+------------
+-- Adjustable Container
+-- (LDoc)
+-- @module Adjustable.Container
```

Comment:
This had to be changed, otherwise generation broke:

```
/home/vadi/Programs/Mudlet/mudlet/src/mudlet-lua/lua/geyser/Geyser.lua:1: contains no items
/home/vadi/Programs/Mudlet/mudlet/src/mudlet-lua/lua/geyser/GeyserAdjustableContainer.lua:17: ?: 'class' cannot have multiple values; {table,module}
```